src/commands/gate/renderer.ts

```diff
@@ -0,0 +1,87 @@
+import chalk from 'chalk'
+
+import {SpanTags} from '../../helpers/interfaces'
+
+import {EvaluationResponse, RuleEvaluation} from './interfaces'
+
+const ICONS = {
+  FAILED: '❌',
+  SUCCESS: '✅',
+  WARNING: '⚠️',
+  INFO: 'ℹ️',
+}
+
+export const renderEvaluationResponse = (evaluationResponse: EvaluationResponse) => {
+  if (evaluationResponse.status.toLowerCase() === 'empty') {
+    return renderEmptyEvaluation()
+  }
+
+  let fullStr = ''
+  fullStr += chalk.green(`Successfully evaluated rules for the current pipeline.\n`)
```

Comment:
nitpick: if there are no variables in the template, there's no need to use "`" 
```suggestion
  fullStr += chalk.green('Successfully evaluated rules for the current pipeline.\n')
```